src/Sylius/Behat/Context/Setup/OrderContext.php

```diff
@@ -119,9 +119,9 @@ public function __construct(
     }
 
     /**
-     * @Given the customer :customer placed an order :orderNumber
+     * @Given there is a customer :customer that placed an order :orderNumber
```

Comment:
`there is a customer` - suggests that we will create a customer, not that this customer already exist. IMO the previous version was fine.